.github/workflows/ci.yml

```diff
@@ -62,6 +62,31 @@ jobs:
           name: checkmk-datasource-signed
           path: '${{ steps.build-signed.outputs.metadata_outputs_archive }}'
 
+  validate-plugin:
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - name: Download package from build job
+        uses: actions/download-artifact@v3
+        with:
+          name: checkmk-datasource-signed
+
+      - name: Setup Go environment
+        uses: actions/setup-go@v3
+        with:
+          go-version: '1.19'
+
+      - name: Install validator
+        run: |
+          git clone https://github.com/grafana/plugin-validator
+          pushd ./plugin-validator/pkg/cmd/plugincheck2
+          go install
```

Comment:
We can use a binary from one of the releases. The tars contain an executable file, a README and a LICENSE, nothing else.

Reply:
this was c&p from the offical actions, but you are right, i adapted it.

Reply:
fine for me

Reply:
does not work, as this file `./plugin-validator/config/default.yaml` is missing. i'm going to remove this commit. we can look into this later